src/lib/tools/Settings.svelte

```
@@ -1,7 +1,7 @@
 <script lang="ts">
 	import { event } from "$lib/utils"
 	import type { GameInstall, SettingsRequest } from "$lib/bindings-types"
-	import { Checkbox, TooltipIcon } from "carbon-components-svelte"
+	import { Checkbox, RadioTile, TileGroup, Tooltip, TooltipIcon } from "carbon-components-svelte"
```

ESLint

[error] 'Tooltip' is defined but never used.

(@typescript-eslint/no-unused-vars)